tools/polkatool/src/main.rs

```diff
@@ -13,6 +13,16 @@ enum Args {
         /// The input file.
         input: PathBuf,
     },
+
+    /// Disassembles a .polkavmm blob into its human-readable assembly.
+    Disassemble {
+        /// The output file.
+        #[clap(short = 'o', long)]
+        output: PathBuf,
```

Comment:
```suggestion
        output: Option<PathBuf>,
```

Wasn't made optional. (:

Reply:
oops haha